deploy/templates/nstemplatetiers/base/ns_dev.yaml

```diff
@@ -57,6 +57,49 @@ objects:
   subjects:
   - kind: User
     name: ${USERNAME}
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
```

Comment:
just to be sure: we can't create a ClusterRole on OSD, can we?

Reply:
We can. 
Are you asking because we would create one ClusterRole that would be referenced by the RoleBindings? We could technically do that, but the creation of the ClusterRole would be done by sandbox-sre. I would rather keep Role & RoleBinding together as there is no way we could modify the reference to a ClusterRole in an existing RoleBinding - that field is immutable.